pkgs/development/python-modules/psycopg2cffi/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, cffi, postgresql, openssl, six }:
+
+buildPythonPackage rec {
+  pname = "psycopg2cffi";
+  version = "2.8.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yc0cxxkfr35kd959wsagxfhy6ikbix71rp9x0rmn858fxa4vapw";
+  };
+
+  buildInputs = [ six ] ++ lib.optional stdenv.isDarwin openssl;
```

Comment:
are six and cffi *only* needed during build-time, not run-time?

Reply:
Same with openssl. With cffi one opens a library, so its important the Python library knows where to find it. Often it needs to be hardcoded.

Reply:
I'm not entirely sure, since for my use case this doesn't currently work. What would I have to change if it were?